engine/src/main/resources/assets/ui/config/autoConfigScreen.ui

```diff
@@ -5,26 +5,42 @@
     "type": "RelativeLayout",
     "contents": [
       {
-        "type": "ColumnLayout",
-        "id": "mainContainer",
-        "fillVerticalSpace": false,
-        "layoutInfo": {
-          "use-content-height": true,
-          "position-horizontal-center": {}
+        "type": "ScrollableArea",
+        "content": {
+          "type": "ColumnLayout",
+          "id": "mainContainer",
+          "fillVerticalSpace": false,
+          "layoutInfo": {
+            "use-content-height": true,
+            "position-horizontal-center": {}
+          },
+          "contents": []
         },
-        "contents": []
+        "layoutInfo": {
+          "width": 600,
```

Comment:
Is this the default? Does it scale in case the window size is decreased?
We switched to a default window size of 8:5 WXGA a while ago, so I believe we can make use of ~ two thirds of that to get a nicely filled yet not weirdly wide look.
But that's no blocker and can be done in a follow-up.

Reply:
Just copied from another screen